tests/integration/components/polaris-resource-list/item-test.js

```diff
@@ -325,10 +351,89 @@ module('Integration | Component | polaris-resource-list/item', function(hooks) {
             }}
           {{/polaris-resource-list/provider}}
         `);
+
         assert.dom('[data-test-checkbox-input]').hasAttribute('disabled');
       });
+
+      test('includes an avatar if one is provided', async function(assert) {
+        await render(hbs`
+          {{#polaris-resource-list/provider value=mockDefaultContext}}
+            {{polaris-resource-list/item
+              id=itemId
+              url=url
+              media=(component "polaris-avatar")
+            }}
+          {{/polaris-resource-list/provider}}
+        `);
+
+        assert.dom(`${mediaSelector} > .Polaris-Avatar`).exists();
+      });
+
+      test('includes a thumbnail if one is provided', async function(assert) {
+        await render(hbs`
+          {{#polaris-resource-list/provider value=mockDefaultContext}}
+            {{polaris-resource-list/item
+              id=itemId
+              url=url
+              media=(component "polaris-thumbnail")
+            }}
+          {{/polaris-resource-list/provider}}
+        `);
+
+        assert.dom(`${mediaSelector} > .Polaris-Thumbnail`).exists();
+      });
     }
   );
 
-  // TODO: add media and shortcutActions tests
+  module(
+    'shortcutActions',
+    {
+      beforeEach() {
+        this.setProperties({ itemId, url, shortcutActions });
+      },
+    },
+    function() {
+      test('does not render shortcut actions if none are provided', async function(assert) {
+        await render(hbs`
+          {{#polaris-resource-list/provider value=mockDefaultContext}}
+            {{polaris-resource-list/item
+              id=itemId
+              url=url
+            }}
+          {{/polaris-resource-list/provider}}
+        `);
+
+        assert.dom(shorcutActionsSelector).doesNotExist();
+      });
+
+      test('renders shortcut actions when some are provided', async function(assert) {
+        await render(hbs`
+          {{#polaris-resource-list/provider value=mockDefaultContext}}
+            {{polaris-resource-list/item
+              id=itemId
+              url=url
+              shortcutActions=shortcutActions
+            }}
+          {{/polaris-resource-list/provider}}
+        `);
+
+        assert.dom(shorcutActionsSelector).exists();
+      });
+
+      test('renders persistent shortcut actions if `persistActions` is true', async function(assert) {
+        await render(hbs`
+          {{#polaris-resource-list/provider value=mockDefaultContext}}
+            {{polaris-resource-list/item
+              persistActions=true
+              id=itemId
+              url=url
+              shortcutActions=shortcutActions
+            }}
+          {{/polaris-resource-list/provider}}
+        `);
+
+        assert.dom('.Polaris-ResourceList-Item__Disclosure').exists();
```

Comment:
`polaris-react` tests for a [`ButtonGroup`](https://github.com/Shopify/polaris-react/blob/master/src/components/ResourceList/components/Item/tests/Item.test.tsx#L372) here, but it looks like our template renders a `polaris-button-group` either way, so I'm checking for `Disclosure` class instead since that's only rendered if `persistActions` is true.

Reply:
Fair point, can you leave this as a comment so that we know why we're differing from the original test here? 👍